commit 1c0e29725fe6bd5210df97fe242cffc671ac8689
Author: Yunlei He <heyunlei@huawei.com>
Date:   Mon Dec 19 20:10:48 2016 +0800

    f2fs: fix a problem of using memory after free
    
    commit 7855eba4d6102f811b6dd142d6c749f53b591fa3 upstream.
    
    This patch fix a problem of using memory after free
    in function __try_merge_extent_node.
    
    Fixes: 0f825ee6e873 ("f2fs: add new interfaces for extent tree")
    Signed-off-by: Yunlei He <heyunlei@huawei.com>
    Reviewed-by: Chao Yu <yuchao0@huawei.com>
    Signed-off-by: Jaegeuk Kim <jaegeuk@kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1338_1c0e29725fe6bd5210df97fe242cffc671ac8689.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1338_1c0e29725fe6bd5210df97fe242cffc671ac8689.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
